lib/pages/chat/input_bar.dart

```diff
@@ -479,3 +375,115 @@ class InputBar extends StatelessWidget {
 class NewLineIntent extends Intent {}
 
 class SubmitLineIntent extends Intent {}
+
+class SuggestionTile extends StatelessWidget {
+  const SuggestionTile({
+    super.key,
+    required this.suggestion,
+    this.client,
+  });
+
+  final Map<String, String?> suggestion;
+  final Client? client;
+
+  @override
+  Widget build(BuildContext context) {
+    const size = 30.0;
```

Comment:
IMO: Not good, create `SuggestionTileStyle`, it's can control all styles of widget